x-pack/plugins/apm/public/utils/testHelpers.tsx

```diff
@@ -116,11 +116,5 @@ export async function getRenderedHref(
 
 export function mockNow(date: string) {
   const fakeNow = new Date(date).getTime();
-  const realDateNow = global.Date.now.bind(global.Date);
-
-  global.Date.now = jest.fn(() => fakeNow);
-
-  return () => {
-    global.Date.now = realDateNow;
-  };
+  return jest.spyOn(Date, 'now').mockReturnValue(fakeNow);
```

Comment:
I made this change, so a standard jest mock is returned. This allows the user to use the built-in methods for clearing mocks like `jest.restoreAllMocks()`.

Reply:
Sounds good. Some day I will remember that jest spies are actually stubs/mocks, but maybe not. I miss sinon, old habits etc. :+1: